### DISCUSSION TITLE
Pagination with GraphQL

### DISCUSSION THREAD

**Discussion by daudln** [uriyyo/fastapi-pagination#1106] [Q&A]
Is it possible to use our library with GraphQL?
Is there any example of how to use?

Stacks:
FastAPI ⚡️, Tortoise ORM 🐢, and Strawberry 🍓

**Comment by uriyyo** [comment]
Hi @daudln,

I guess it should be possible, I need to take a look at `strawberry`.

**Reply by daudln** [reply]
Nice to here @uriyyo
